riptable/rt_timezone.py

```diff
@@ -350,7 +350,13 @@ class TimeZone:
         'NYC'    : 'America/New_York',
         'DUBLIN' : 'Europe/Dublin',
         'GMT'    : 'GMT',
-        'UTC'    : 'GMT'
+        'UTC'    : 'UTC'

```

Comment:
While I know there is no difference between GMT and UTC, this does mean that all timezone representations that used to be 'GMT' will now render as 'UTC'. Not sure if there could be any fallout from that (i.e. anyone does string compare, sorting, etc?